nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -1245,6 +1243,187 @@ def __parse_item_rttm(self, line: str, manifest_file: str) -> Dict[str, Any]:
         return item
 
 
+class EndtoEndDiarizationLabel(_Collection):
+    """List of diarization audio-label correspondence with preprocessing."""
```

Comment:
These oneliner docstrings are not updated when copied from the original source. Update it.

Reply:
Fixed and updated